src/main/java/com/jland/repository/impl/ConferenceRepositoryImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.jland.repository.impl;
```

Comment:
I think we can use Spring Data instead of jdbc directly. Do you have any concern? Can agree in our chat